homeassistant/components/v2c/__init__.py

```diff
@@ -20,6 +21,13 @@
 ]
 
 
+@dataclass
```

Comment:
The class is unnecessary 
You can just put the coordinator in the runtime data

Reply:
fixed